docker-compose.yml

```diff
@@ -98,6 +132,26 @@ services:
       bob:
         condition: service_healthy
 
+  tunnel-tails:
+    image: wernight/ngrok
+    command: ngrok http tails:6543 --bind-tls false --log stdout
+
+  tails:
+    image: ghcr.io/bcgov/tails-server:latest
+    ports:
+      - 6543:6543
+    environment:
+      - GENESIS_URL=https://raw.githubusercontent.com/Indicio-tech/indicio-network/master/genesis_files/pool_transactions_testnet_genesis
```

Comment:
This is currently out of sync with the networks being used by the Alice and Bob services; switching the alice and bob services to use testnet will fix this.